### DISCUSSION TITLE
horizontal scrolling broken

### DISCUSSION THREAD

**Discussion by 3052** [community/community#112807] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

example URL:

https://github.com/3052/media/blob/main/mubi/secure.json

**Comment by ghost** [comment]
> ### Select Topic Area
> Bug
> 
> ### Body
> example URL:
> 
> https://github.com/3052/media/blob/main/mubi/secure.json

It's fine

**Comment by taylor-true** [comment]
It works fine for me too. Typical "turn it off then turn it back on" suggestion:
- Switch browsers 
- Check if your horizontal scroll works on other pages

**Reply by ghost** [reply]
Still same , works well ...

**Reply by ghost** [reply]
Now i got something the solution is you have to scroll to the end then again scroll back 
this will fix it

**Reply by 3052** [reply]
just tried again with a clean install, still broken. does work here:
https://codepen.io/jaemskyle/pen/XWJvRL

**Comment by Ik0ri4n** [comment]
Since this discussion is quite vague, I will try to add some insights and hope that it fits the context of this discussion. If not please ping me and I may move that to a separate page.

## Horizontal scrolling with mouse via scrollbar

The GitHub code view page implements a custom scrollbar which replaces the default scroll behavior and does not handle scrolling quickly to the end of the line correctly. This bug occurs on Firefox but not Chromium for me (likely because of different support for custom scrollbars).

An example of what this looks like:
![Screenshot from 2024-04-16 10-29-38](https://github.com/community/community/assets/48494824/99eded3c-2059-4eaf-87d5-119725925aec)
![Screenshot from 2024-04-16 10-29-54](https://github.com/community/community/assets/48494824/3849690d-620c-4a67-bd76-6a6bf4adb893)

As you can see, quite a bit of text can be unreachable. How much seems to depend on the scroll event resolution and thus on the scroll speed. It is pretty annoying though if you want to look up code quickly!
